src/scripts/farming/Farming.ts

```diff
@@ -1737,6 +1737,11 @@ class Farming implements Feature {
 
         //#endregion
 
+        // REMOVE THIS BEFORE MERGING. THIS IS FOR TESTING PURPOSE.
+        this.berryData.forEach(b => {
+            const witherTime = b.growthTime[PlotStage.Berry] - b.growthTime[PlotStage.Bloom];
+            b.growthTime = [30, 60, 90, 120, 120 + witherTime];
+        });
```

Comment:
Just commenting here so it's not forgotten 😓

Reply:
Removed, thanks for the reminder !